src/main/java/com/mozi/lotto/LottoMachine.java

```diff
@@ -0,0 +1,59 @@
+package com.mozi.lotto;
+
+public class LottoMachine {
+    private int pay;
+    private int numberOfGames;
+    private int manualInputCount;   // 수동 입력 개수 저장
+
+    public LottoMachine(int pay, int manualInputCount) {
+        this.pay = pay;
+        this.numberOfGames = pay / 1000;
+        if (manualInputCount > this.numberOfGames) {
+            throw new IllegalArgumentException("[ERROR] 돈이 부족합니다!");
+        }
+        if (manualInputCount < 0) {
+            throw new IllegalArgumentException("[ERROR] 제대로 입력해 주세요!");
+        }
+        this.manualInputCount = manualInputCount;
+    }
+
+    public int getPay() {
+        return pay;
+    }
+    public int getNumberOfGames() {
+        return numberOfGames;
+    }
+
+    public int getManualInputCount() {
+        return manualInputCount;
+    }
+
+    public LottoTicket generateManualTicket() {
+        LottoTicket lottoTicket = new LottoTicket();
+        System.out.println("수동으로 구매할 번호를 입력해 주세요. (1, 2, .. , 6): ");
+        for (int i = 0; i < this.manualInputCount; i++) {
+            lottoTicket.getTicket().add(InputHandler.getValidatedManualNumber());
```

Comment:
`LottoTicket` 오브젝트 내부의 리스트를 꺼내와서 직접 `add`하는 것보다, 오브젝트에게 메서드를 통해 전달하고 그 메서드 안에서 직접 추가하게 하는 것이 좋습니다.
```suggestion
            lottoTicket.addLottoNumbers(new LottoNumbers(NumberGenerator.generateNumber()));
```